backoffice/www/server.py

```diff
@@ -0,0 +1,58 @@
+import os.path
+
+import tornado.ioloop
+import tornado.locale
+import tornado.web
+from absl import logging
+from sqlalchemy import create_engine
+from tornado_sqlalchemy import SQLAlchemy
+
+from backoffice.model.user import Base
+from backoffice.www.handlers.home import HomeBOHandler
+from backoffice.www.handlers.login import LoginHandler
+from backoffice.www.handlers.user import ListUserHandler, UserJson, CreateUserHandler
+
+
+class BackOfficeServer:
+  """Serves and manipulates the Backoffice ICUBAM."""
+
+  def __init__(self, config, port):
+    self.config = config
+    self.port = port
+    self.routes = []
+    self.db = SQLAlchemy("sqlite+pysqlite:///{}".format(self.config.db.sqlite_path))
+    self.make_app()
+
+  def add_handler(self, handler, **kwargs):
+    route = os.path.join("/", handler.ROUTE)
+    self.routes.append((route, handler, kwargs))
+    logging.info("{} serving on {}".format(handler.__name__, route))
+
+  def make_app(self):
+    self.add_handler(LoginHandler)
+    self.add_handler(ListUserHandler)
+    self.add_handler(UserJson)
+    self.add_handler(CreateUserHandler)
+    self.add_handler(HomeBOHandler)
+
+  def run(self):
+    logging.info(
+      "Running {} on port {}".format(self.__class__.__name__, self.port)
+    )
+    engine = create_engine("sqlite+pysqlite:///{}".format(self.config.db.sqlite_path), encoding='utf-8', echo=True)
```

Comment:
can we have in the constructor:

self.config ....
...
self.sqlite_path = "sqlite+pysqlite:///{}".format(self.config.db.sqlite_path)

then we can use it to initialise self.db (line 23) and the engine here.